k8s/helm/values.yaml

```diff
@@ -8,6 +8,9 @@ cesLabels: &cesLabels
   *cloudoguPartOfLabelKey: *cloudoguPartOfLabelValue
 
 velero:
+  image:
+    imagePullSecrets:
+      - "ces-container-registries"
```

Comment:
Should we set `.velero.serviceAccount.server.imagePullSecrets` as well?

Reply:
Where did you get this key? I don't find it. Nevertheless setting `image.imagepullSecrets` should be okay.

Reply:
This is a newer key not yet supported by the version used by us.